jupyter/jupyter-web-app/overlays/application/kustomization.yaml

```diff
@@ -4,9 +4,22 @@ bases:
 - ../../base
 resources:
 - application.yaml
+configMapGenerator:
+- name: jupyter-web-app-parameters
+  env: params.env
+vars:
+- name: generateName
+  objref:
+    kind: ConfigMap
+    name: jupyter-web-app-parameters
+    apiVersion: v1
+  fieldref:
+    fieldpath: data.generateName
+configurations:
+- params.yaml
 commonLabels:
   app.kubernetes.io/name: jupyter-web-app
-  app.kubernetes.io/instance: jupyter-web-app
+  app.kubernetes.io/instance: $(generateName)
```

Comment:
Why are we parameterizing the instance name rather than just explicitly coding it here in common labels?